packages/core-mobile/app/utils/navigation/Stack.tsx

```diff
@@ -0,0 +1,16 @@
+import { ParamListBase, StackNavigationState } from '@react-navigation/native'
+import {
+  createStackNavigator,
+  StackNavigationEventMap,
+  StackNavigationOptions
+} from '@react-navigation/stack'
+import { withLayoutContext } from 'expo-router'
+
+const { Navigator } = createStackNavigator()
+
```

Comment:
these 2 files I think should be moved to the `new` folder since they are only used in there

Reply:
oh, but the `app/new` folder should have files only for screens and layouts used by `expo-router`. if we want to use `app/new` more generic, I think we should add another sub-folder in it and move those files into it.